docs/docs/20-usage/40-secrets.md

```diff
@@ -4,6 +4,15 @@ Woodpecker provides the ability to store named parameters external to the YAML c
 
 Secrets are exposed to your pipeline steps and plugins as uppercase environment variables and can therefore be referenced in the commands section of your pipeline.
 
+Woodpecker provides three different levels to add secrets to your pipeline. The following list shows the priority of the different levels. If a secret is defined in multiple levels, will be used following this priorities: Repository secrets > Organization secrets > Global secrets.
+
+1. **Repository secrets**: They are available to all pipelines of an repository.
+1. **Organization secrets**: They are available to all pipelines of an organization.
+1. **Global secrets**: Can be configured by an instance admin.
```

Comment:
It does not affect rendering but is easier to read in source code:

```suggestion
2. **Organization secrets**: They are available to all pipelines of an organization.
3. **Global secrets**: Can be configured by an instance admin.
```

Reply:
well MD will render it valide ... but I think it's nice to have it explicite

Reply:
I know markdown will render them correctly but it's still hard to look at them in editor